resources/assets/js/components/FriendDetails.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <button type="button" class="btn btn-primary btn-xs" @click="showDetails(friend)">Details</button>
+
+    <div id="{{ friend.username }}" class="hide">
+        <div class="friend-details-modal">
+            <img v-bind:src="friend.avatar_url" class="img-circle" height="160" />
```

Comment:
No need to have closing `/` on an image tag, since we're in HTML instead of XHTML these days.